frontend/views/containers/chatroom/ChatNotificationSettingsModal.vue

```diff
@@ -0,0 +1,129 @@
+<template lang='pug'>
+  modal-template(ref='modal' :a11yTitle='L("Notifications")')
+    template(slot='title')
+      i18n Notifications
+
+    form(novalidate @submit.prevent='' data-test='updateNotificationSettings')
+      fieldset.is-column
+        legend.legend Send notifications for:
+        label.radio
+          input.input(
+            type='radio'
+            name='messageNotification'
+            :value='options.ALL_MESSAGES'
+            v-model='$v.form.messageNotification.$model'
+          )
+          span All new messages
```

Comment:
@Silver-IT is it alright not to translate sentence here (also other sentences below) using `i18n`?

Reply:
Nice catch. I think I missed that part. We should use `i18n`.